utils/docker/build.sh

```diff
@@ -25,14 +25,18 @@ function docker_build
 }
 
 # Build all images.
-docker_build Dockerfile.base            ubuntu-base
-docker_build Dockerfile.ci-image-base   ci-image-base
-docker_build Dockerfile.build-clang     ubuntu-build-clang  \
+docker_build Dockerfile.base                      ubuntu-base
+docker_build Dockerfile.ci-image-base             ci-image-base
+docker_build Dockerfile.build-clang               ubuntu-build-clang              \
     --build-arg "BASE=${NAME}:ubuntu-base"
-docker_build Dockerfile.build-clang     latest              \
+docker_build Dockerfile.build-clang               latest                          \
     --build-arg "BASE=${NAME}:ubuntu-base"
-docker_build Dockerfile.build-gcc       ubuntu-build-gcc    \
+docker_build Dockerfile.build-gcc                 ubuntu-build-gcc                \
     --build-arg "BASE=${NAME}:ubuntu-base"
+docker_build Dockerfile.build-clang-plugins-deps  ubuntu-build-clang-plugins-deps \
+    --build-arg "BASE=${NAME}:ubuntu-build-clang"
+docker_build Dockerfile.build-gcc-plugins-deps    ubuntu-build-gcc-plugins-deps   \
+    --build-arg "BASE=${NAME}:ubuntu-build-gcc"
```

Comment:
You can only have one new file with changing the `BASE` image. I found the both files are almost the same.